CV/Official_Announcement.md

```diff
@@ -45,16 +45,18 @@ Riêng buổi học cuối cùng, sau khi giờ kết thúc, BTC dự kiến s
 * AI Solutions Architect
 * Tốt nghiệp Thạc sĩ, Đại học Tokyo
 
-And more...!
+(Danh sách người hướng dẫn sẽ liên tục được cập nhật trong thời gian tới)
+
+Khối lượng công việc cần chuẩn bị và hướng dẫn khoá học khá lớn. BTC rất mong các thành viên có kiến thức và kinh nghiệm về Computer Vision hoặc Deep Learning cùng tham gia hướng dẫn khoá học này cùng chúng tôi. Thành viên muốn tham gia hướng dẫn có thể liên hệ với 3 thành viên trên, hoặc comment vào post này.
 
 
 ## Điều kiện tham gia
-* Là người có background về lập trình, đặc biệt là python, nhưng chưa có nhiều kiến thức và kỹ năng về ML/DL
-* Là người muốn học hỏi và tìm hiểu lĩnh vực ML/DL
+* Là người có background về lập trình, đặc biệt là python, đồng thời là người mới bắt đầu học về Machine Learning/Deep Learning (ML/DL)
+* Là người muốn học hỏi thêm và tìm hiểu sâu về lĩnh vực ML/DL
 
 
 ## Thông tin về tài liệu chuẩn bị trước khi tham gia course
-Dưới đây là 1 số tài liệu gợi ý giúp các bạn muốn tham gia khoá học nhưng chưa có đủ kỹ năng lập trình Python và xử lý dữ liệu với Numpy có thể chuẩn bị trước những kỹ năng này.
+Dưới đây là 1 số tài liệu gợi ý giúp các bạn muốn tham gia khoá học nhưng chưa có đủ kỹ năng lập trình Python và xử lý dữ liệu với Numpy có thể học trước để chuẩn bị.
 
 1. Basic on Python:
 * https://www.programiz.com/python-programming/first-program
```

Comment:
Có cần chữ "tượng trưng" sau học phí không a? Em thấy không cần thiết, cái đấy mình nói rõ là để chi vào việc gì rồi.

Reply:
ImageNet có được gọi là thuật toán hay networks đâu nhỉ?

Reply:
Thay ImageNet chỗ này bằng ResNet nhé